source/2019-08-15-octane-release-plan.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Octane is coming in v3.14
+author: Yehuda Katz, Jen Weber
+tags: 2019, Announcement, Recent Posts
+responsive: true
+---
+
+The Ember community is wrapping up the [Octane](https://emberjs.com/editions/) edition, and we expect to get it over the finish line over the next three releases!
+
```

Comment:
I think it would be helpful to have a one- or two-sentence description of Octane near the introduction. The post currently dives in assuming people already know what Octane is.

Reply:
>The Ember community is wrapping up the work for [Octane](https://emberjs.com/editions/), and we expect to get it over the finish line over the next three releases!
Ember Octane updates Ember with modern, streamlined components and stage management that makes it fun to build components. It has seamless interoperability with existing apps, letting teams migrate at their own pace.

Reply:
This is very good feedback, thank you!